### DISCUSSION TITLE
Ukraine Liste

### DISCUSSION THREAD

**Discussion by RPiList** [RPiList/specials#1129] [Ideas]
Hi,

ich habe eine Anfrage erhalten, bzgl. einer Blockliste die Unternehmen/Marken blockt, die immer noch in Russland "arbeiten". 
Hintergrund sind wohl solche Berichte: https://orf.at/stories/3322611/

https://www.theguardian.com/business/2023/jul/03/unilever-named-international-sponsor-of-war-by-ukraine

Auf der einen Seite wäre solch eine Liste natürlich nur symbolisch. Ich kaufe das Produkt XYZ ja nicht direkt beim Hersteller/Unilever sondern i.d.R. beim Rewe/Edeka/Aldi um die Ecke. 

Auf der anderen Seite hat so eine Liste evtl. einfach nur eine Signal-Wirkung. Es ist eine Art von "Shaming" die Unternehmen oftmals vermeiden wollen.

Da die Nutzung solch einer Liste freiwillig ist, sehe ich keinen großen Schaden, wenn wir so etwas anbieten. Ist so ähnlich wie die sog. "Springer"-Liste. Die haben wir hier zwar nicht erstellt, brauchten wir aber auch nicht, weil andere das schon anbieten.

Schwieriger ist da schon die Definition der Domains die auf die Liste kommen. Kommen da nur Unternehmen drauf die in Russland "arbeiten". Oder auch andere, die Russland unterstützen... usw usf...

Also da wird es Entscheidungen geben müssen, wann wer auf die Liste kommt. Wobei das evtl. kein so großes Problem ist. Wenn wir für einen Nutzer übers Ziel hinaus schießen, kann man unsere Liste "forken" und das als "Light-Version" selbst anbieten.

Fangen wir also an; mit einem Namen für die Liste:

Russland
Ukraine
WarProfiteer
...

**Comment by BobderNixkoenner** [comment]
Um mich auch mal etwas nützlich zu machen,
habe ich mit einem Python-Skript die Top 100 oder so (von insgesamt 1380) Unternehmen mit Stand vom 06.07.2023 ausgelesen. 
Hier ist die Liste:

Philip Morris
Japan Tobacco International
Leroy Merlin
PepsiCo
Auchan
Metro AG
Nestle
Danone
Mars
Procter & Gamble
Chery Automobile
Cargill
Raiffeisen Bank
Anheuser-Busch InBev
Mondi
Mondelez
Haier
Unilever
Haval Motor
Globus
Komatsu
Schlumberger (SLB)
Hayat Holding
Geely
L´Oreal
Rönesans Holding
Syngenta
Ferrero
Krka
Stada
Intesa Sanpaolo
Jacobs Douwe Egberts
Tarkett
Liebherr
Claas Group
Sisecam
Westinghouse
Egger
Teva
Charoen Pokphand Foods
Gorenje
Bacardi
Barloworld
Beko
China Railway Construction Corporation
Berlin-Chemie
FAW Group
GlaxoSmithKline
Zeppelin
Calzedonia
Rockwool
Dr. Reddy & Laboratories
Sany Heavy Industries
TAPON FRANCE
FM Logistic
Mitsui
Xiaomi
Weatherford International
B. Braun
Parmalat
Makita
Perfetti Van Melle
Hyosung
LACTALIS
Candy
PVH
Pioneer
Fresenius
Alibaba
Bonduelle
Kärcher
Paldo Food Co
Döhler
Siemens Healthineers
Strauss Coffee
Selgros
Ontex Group
Storck
Amazone
Hikvision
Future Group
New Yorker Group Services
EOS
Frigoglass
Froneri
Highland Gold
Wanhua Chemical Group
Midea Group
ALPLA
Egis
Titan International
Ariston
Mansour Group
Wilo SE
Enka İnşaat ve Sanayi A.Ş.
Armstrong World Industries
Merz Pharma
Verallia Deutschland
BDR Thermea
Benetton
FCB Holding Cooperatief
Lesaffre
Orange Business Services
Puratos
Sun Pharma
Chipita
GEA Refrigeration Technologies
DP World
IG Group
PERI Group
Vivo
Imperial Energy Corporation
China Huadian Corporation
Evyap
Dufry
Campari Group
Ritter Sport
Bentec
Huntsman Corporation
Groupe Savencia


Grundsätzlich muss ich sagen, dass ich eine solche Liste nützlich finde. 
Allerdings ist mir klar dass viele dieser Unternehmen auch Tochterfirmen haben. 
Außerdem gibt es auch Firmen, die einen dieser Namen in ihrer eigenen Domain haben, aber selbst nicht direkt mit der genannten Firma in Verbindung stehen, außer dass sie möglicherweise ein Franchise-Unternehmen sind. 
Ein Beispiel hierfür ist "Kärchershop-Schreiber" (kenne ich nicht, habe nur davon gehört).
Die Pflege einer solchen Liste, selbst in einer abgespeckten Version, dürfte sehr arbeitsintensiv sein. 
Eine Idee (aber keine gute) wäre es, nur deutsche Unternehmen in die Liste aufzunehmen, um weniger Arbeit zu haben. 
Allerdings würde man dabei auch wiederum Unternehmen wie Philip Morris, Mars oder Nestle verpassen, die meiner Meinung nach ohnehin auf einer solchen Liste stehen sollten.

Es erscheint mir unwahrscheinlich, eine Liste zu erstellen, die auch die Tochterfirmen enthält.

Um es uns bzw. dir etwas einfacher zu machen, könnten wir doch den Betreiber der Website kontaktieren und darum bitten, uns die Listen zukommen zu lassen. 
Immerhin verfügen sie über die Listen der 1380 Unternehmen inklusive der dazugehörigen Websites. 
Es dürfte in ihrem Interesse liegen, wenn wir bzw. du eine Liste erstellst, die mit Pi-hole und AdGuard kompatibel ist. 
Dadurch würden sie Aufmerksamkeit für ihre Website in der Pi-hole- und AdGuard-Gemeinschaft generieren. 
Und falls die einen guten Tag haben, könnten sie auch Werbung für unsere bzw. deine Liste machen, was gleichzeitig Werbung für den besagten Kaltgetränkevernichter ist. 
Ein Win-Win-Szenario also.


In diesem Sinne

**Comment by RPiList** [comment]
Name der Liste ist "Still-Supporting-Russia" bis uns was besseres einfällt.

Tochter-Gesellschaften werden in der Regel auch gebannt, wenn die in RU noch tätig sind. Im Umkehrschluss wird ne Tochter-Ges nicht gebannt, wenn sie sich zurückgezogen hat. 

Die Organisatoren der Website können wir gerne ansprechen aber an deren Liste zu kommen ist auch so kein Problem. Das "Problem" ist an die Marken zu kommen, unter denen die Unternehmen auftreten. Pepsico hat sicher 50 Marken... da hoffe ich jetzt mal auf Wikipedia. Procter & Gamble ist eines der größten Unternehmen der Welt... die haben an die 100 Marken und die dann für fast jedes Land der Erde.

Insoweit wird die Liste sicher erst nach dem Krieg fertig sein. :-) Aber wir haben es ja auch nicht eilig. Online gehen kann die Liste sobald die ersten Domains drauf sind. Wird dann nach und nach erweitert.

Wer kommt auf die Liste: Für unsere Liste ist die Welt schwarz weiß. Macht ein/e Unternehmen/Marke Geschäfte in RU, unterstützt es RU und kommt auf die Liste. Klar kann man argumentieren, das China-Unternehmen nie gesagt haben, dass sie gegen RU sind, oder sie neutral seien. Aber in einer Schwarz/Weiß-Welt ist man für Russland oder dagegen.
